### PR TITLE
[MRG] Fix yarn lint errors for build to pass.

### PR DESCRIPTION
**Pull request by sharanry**
#32
**FIXED** 
- [x] import/first
- [x] no-unused-vars
- [x] class-methods-use-this
- [x] max-len
- [x] no-tabs
- [x] no-multi-spaces
- [x] no-mixed-spaces-and-tabs
- [x] react/jsx-no-bind
- [x] react/prop-types
- [x] react/forbid-prop-types
- [x] react/require-default-props

### CODE REVIEW

**Comment by prampey** [comment]
Awesome! Can't wait for all builds to pass.

**Comment by prampey** [comment]
The build seems to be failing for some reason. @SebastinSanty Can you take a look?

**Comment by prampey** [comment]
Looks like you've added the _react/forbid-prop-types_ rule under `tools/.eslintrc`. This config file is meant only to override ESLinting in all the build scripts present in that particular `tools` directory. Any **general** rule changes must be reflected under the `eslintConfig` option in 'package.json'.

File: tools/.eslintrc [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {
+  "plugins": {
```
Comment (prampey) [inline tools/.eslintrc]: This section is not required here. Remove it.

File: tools/.eslintrc [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+  	"react",
+  	"no-console",
+  	"global-require"
+  },
   "rules": {
     "no-console": "off",
-    "global-require": "off"
+    "global-require": "off",
+    "react/forbid-prop-types": 0,
```
Comment (prampey) [inline tools/.eslintrc]: `forbid-prop-types` needs to be added under `esLintconfig` in `package.json`

**Comment by sharanry** [comment]
@prampey I made the changes you had asked but now its showing a new range errors all related to components/Footer/footer.css and components/variables.css. They are quite a few of them

**Comment by sharanry** [comment]
@prampey @SebastinSanty The build is finally passing, could you review the PR once?

**Comment by prampey** [comment]
Please squash commits before merging.

**Comment by SebastinSanty** [comment]
I squashed them.

**Pull request merged**